prql-java/src/lib.rs

```diff
@@ -1,24 +1,64 @@
 use jni::objects::{JClass, JString};
-use jni::sys::jstring;
+use jni::sys::{jboolean, jstring};
 use jni::JNIEnv;
-use prql_compiler::{json, prql_to_pl, Options};
+use prql_compiler::sql::Dialect;
+use prql_compiler::{json, pl_to_prql, prql_to_pl, ErrorMessages, Options, Target};
 
 #[no_mangle]
 #[allow(non_snake_case)]
 pub extern "system" fn Java_org_prql_prql4j_PrqlCompiler_toSql(
     env: JNIEnv,
     _class: JClass,
     query: JString,
+    dialect: JString,
```

Comment:
We should merge because this is an improvement, but this really should be `target`, and it should be matching against strings like `sql.bigquery`, rather than `bigquery`, from https://github.com/PRQL/prql/blob/main/book/src/language-features/target.md#L10-L11

Reply:
Fixed.